docs/docs/examples/emojivoto.md

```diff
@@ -17,37 +17,25 @@ votes to the frontend.
 
 ![emojivoto components topology](https://raw.githubusercontent.com/BuoyantIO/emojivoto/e490d5789086e75933a474b22f9723fbfa0b29ba/assets/emojivoto-topology.png)
 
-### Motivation
-
-Using a voting service, users' votes are considered highly sensitive data, as we require
-a secret ballot. Also, users are likely interested in the fairness of the ballot. For
-both requirements, we can use Confidential Computing and, specifically, workload attestation
-to prove to those interested in voting that the app is running in a protected environment
-where their votes are processed without leaking to the platform provider or workload owner.
-
```

Comment:
Maybe we could keep a shorter (or alternate) version of this somewhere? I think making the goal of verification explicit helps understanding the purpose of it.

Reply:
I added some explanation above.